src/__tests__/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
I don't think this file is actually needed anymore if you remove the `exclude` from the root `tsconfig.json`